File: swift/kneeso.swift
let width = Int(readLine()!)!
let length = Int(readLine()!)!
let pattern = ["R", "W"];
var str = ""
for (var idx = 0; idx < length; idx++) {
    let colorStr = pattern[(idx / width) % pattern.count]
    str += colorStr
}
print(str)

File: cpp/kneeso.cpp
#include <iostream>
#include <string>
#include <cstdlib>
#include <vector>
using namespace std;
string readline() {
    string str;
    getline(cin, str);
    return str;
}
int main(void){
    auto width = stoi(readline());
    auto length = stoi(readline());
    
    vector<string> pattern = {"R", "W"};
    
    string str = "";
    for (auto idx = 0; idx < length; idx++) {
        str += pattern[(idx / width) % pattern.size()];
    }
    cout << str << endl;
    
    return EXIT_SUCCESS;
}
